components/post_view/post_edited_indicator/post_edited_indicator.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import Icon from '@mattermost/compass-components/foundations/icon/Icon';
+
+interface Props {
+    postId?: string | null;
+    editedAt?: number;
+}
+
+const PostEditedIndicator = ({postId = null, editedAt = 0}: Props): JSX.Element | null => {
+    return !postId || editedAt === 0 ? null : (
```

Comment:
0/5 this is a personal taste, but I would prefer this to be in the form of
```
if CONDITION {
  return null
}
return (COMPONENT)
```

Nevertheless, this is just my personal taste. Not sure if we have other style guidelines that prefer the `x ? a : b` approach.

Reply:
yep , that is one of the changes I just made to the component